content/guides/hot-module-replacement.md

```diff
@@ -18,125 +19,194 @@ related:
     url: /api/hot-module-replacement
 ---
 
+T> This guide extends on code examples found in the [`Development`](/guides/development) guide.
```

Comment:
Can you remove the backticks from "[`Development`]"? We use them for code, but decided to not use them for links in the guides.